Elsewhen/Helpers/TimeZoneFlags.swift

```diff
@@ -881,3 +881,409 @@ let timeZoneCountries: [String: String] = [
     "Pacific/Wake" : "United States Minor Outlying Islands",
     "Pacific/Wallis" : "Wallis and Futuna",
 ]
+
+let alternativeTimeZoneSearchTerms: [String: [String]] = [
+//    "Africa/Abidjan" : ["Ivory Coast, Côte d'Ivoire"],
```

Comment:
Why are all these commented out? 😁

Reply:
Because they're a copy-paste of the "countries" one above, with the ones I've actually changed to being "other terms to search for" uncommented! It was easier to leave them in so I know which time zones I *haven't* added additional search terms for rather than having to diff two lists each time 😅 unconventional, I know …

Reply:
Fair enough!